Model/Sales/Repository/PackageRepository.php

```diff
@@ -56,27 +61,40 @@ public function getPackageType()
      */
     public function selectPackageType(array $products): string
     {
-        $packageTypes = AbstractConsignment::PACKAGE_TYPES_NAMES;
-        $package      = AbstractConsignment::PACKAGE_TYPE_PACKAGE_NAME;
+        $packageType = [];
 
         if ($this->isMailboxActive() || $this->isDigitalStampActive()) {
             foreach ($products as $product) {
-                foreach ($packageTypes as $packageType) {
```

Comment:
Het is beter om hier 2 loopjes te maken voor beide package types

Reply:
Voor beide package types eigen functies gebruiken. Als ze veel op elkaar lijken zou je die abstract kunne maken.

Ik zie dat jullie vaak $is... variabelen gebruiken. Dat is eigenlijk altijd een teken dat er een functie voor gemaakt moet worden. Variabelen bepalen meestal een waarde, terwijl functie een gedrag bepalen.